test/unit_Polygon.jl

```diff
@@ -280,6 +280,12 @@ for N in [Float64, Float32, Rational{Int}]
     @test σ(d, vp) == points[1]
     d = N[0, -1]
     @test σ(d, vp) == points[2]
+    dirs = [[1, 0], [1, 0.5], [0.5, 0.5], [0.5, 1], [0, 1], [-0.5, 1], [-0.5, 0.5], [-1, 0.5], [-1, 0], [1, -0.5], [0.5, -0.5], [0.5, -1], [0, -1], [-0.5, -1], [-0.5, -0.5], [-1, -0.5]]
+    B = Ball2(zeros(2), 1.0)
+    P = HPolygon([HalfSpace(di, ρ(di, B)) for di in dirs])
+    for (i, d) in enumerate(vertices_list(P))
+        @test σ(d, P) == P.vertices[i]
```

Comment:
This will probably not work (if it works, it will be extreme luck) because every `σ` query has two possible answers (the polygon's edges are orthogonal to the directions). You have to rotate the polygon a bit. Maybe you can rotate the directions when creating `P`?

Reply:
I just saw that you use the vertex `d` as the direction, which in this special case is fine :+1:
So you can undo the latest change if you want (sorry).